frontend/src/pages/Exercise/index.js

```diff
@@ -208,13 +217,15 @@ export class Exercise extends Component<Props> {
                       <Number
                         className={css(`width: 100%`)}
                         label={item.question}
-                        min={0}
+                        min={15}
```

Comment:
I think keep this as `0` because when you come to the question, it already says 15 and so I can just click next without adding my actual age